src/Monolog/Formatter/LogzIoFormatter.php

```diff
@@ -0,0 +1,50 @@
+<?php declare(strict_types=1);
+
+/*
+ * This file is part of the Monolog package.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Monolog\Formatter;
+
+/**
+ * Encodes message information into JSON in a format compatible with Logz.io.
+ *
+ * @author Christian Brückner <chris@chrico.info>
+ */
+class LogzIoFormatter extends JsonFormatter
+{
+
+    /**
+     * yyyy-MM-dd'T'HH:mm:ss.SSSZ
+     */
+    const DATETIME_FORMAT = "c";
```

Comment:
Why don't you include microseconds (i.g. `Y-m-d\TH:i:s.uP`)? Personally I prefer (auto-)ordering my log-trace based on time (especially in the ELK-stack). 

```php
$dt = new DateTime('now');

echo $dt->format('c');
echo $dt->format('Y-m-d\TH:i:s.uP');

----
2017-06-29T11:55:49+00:00
2017-06-29T11:55:49.763333+00:00
```